src/main/tmdb/directives/awesomeSearch.js

```diff
@@ -0,0 +1,33 @@
+/**
+ * Provides a search element
+ *
+ * @module tmdb.directives.awesomeSearch
+ *
+ * @requires angular
+ * @requires SearchController
+ *
+ * @author John Gomez <jgomez@alertlogic.com>
+ *
+ * @copyright Alert Logic, Inc 2016
+ *
+ */
+
+ define( [ 'angular', 
+           'tmdb/partials/awesomeSearch/AwesomeSearchController' ], 
+    function( angular, AwesomeSearchController ) {
+        "use strict";
+
+        return function() {
+            return {
+                transclude: true,
+                replace: true,
+                controller: AwesomeSearchController,
+                templateUrl: '/tmdb/partials/awesomeSearch/awesomeSearch.html',
+                restrict: 'E',
+                scope: {
+                    movieList: '=ngModel'
```

Comment:
I think this was probably copied from some of the original source code -- I haven't figured out why scope is bound this way yet.